app/javascript/components/pages/Portfolio.js

```diff
@@ -1,11 +1,18 @@
 import React, { Component } from "react";
-import { Card, Row, Col, CardText, CardTitle } from "reactstrap";
-import { Link } from "react-router-dom";
+import { Card, Row, Col, CardText, CardTitle, Dropdown, DropdownToggle, DropdownMenu, DropdownItem, Button, Modal, ModalHeader, ModalBody, ModalFooter, Form, FormGroup, Input, Label} from "reactstrap";
+// import { Modal, Button } from "react-bootstrap";
```

Comment:
Looks like you already handled this import.

Reply:
thanks removed that line